app/mobile/lib/services/annotationService.dart

```diff
@@ -5,7 +5,7 @@ import 'package:get_it/get_it.dart';
 class AnnotationService {
   static final Dio _dio = Dio();
   static final GetIt getIt = GetIt.instance;
-  static String baseUrl = 'http://34.29.3.88:1938/';
+  static String baseUrl = 'http://3.72.76.16:1938/';
 
   // static void setup() {
```

Comment:
remove commented out code

Reply:
those comments are left there for future improvements. important ones.